src/main/webapp/js/components/csv/csv-config-page.jsx

```diff
@@ -218,6 +227,7 @@ class CsvConfigPage extends Component {
                                           <button
                                             type="button"
                                             className="btn btn-danger"
+                                            disabled={fieldConfig ? fieldConfig.required : false}
```

Comment:
this is not removing the field config, just the value map for enum fields and should not be disabled